static/js/controllers/reports.js

```diff
@@ -250,17 +336,18 @@ var _ = require('underscore'),
             _query();
           }
         }
-
       };
 
       $scope.$on('ClearSelected', function() {
```

Comment:
oh, I see, `$scope.clearSelected` belongs to inbox. Got it.
I should get around to https://github.com/medic/medic-webapp/issues/2133 so that I stop doing these thought loops over and over